doc/operation/performance_tuning_tips.md

```diff
@@ -1,6 +1,40 @@
-# Tuning tips for increasing the performance
+#<a name="top"></a>Tuning tips for increasing the performance
+Content:
 
-## Sink parallelization
+* [Batching](#section1)
+* [Sink parallelization](#section2)
+    * [Multiple sinks, single channel](#section2.1)
+    * [Multiple sinks, multiple channels](#section2.2)
+    * [Why the `LoadBalancingSinkProcessor` is not suitable](#section2.3)
+* [Channel considerations](#section3)
+    * [Channel type](#section3.1)
+    * [Channel capacity](#section3.2)
+* [Events TTL](#section4)
+* [Grouping rules](#section5)
+* [Writing logs](#section6)
+* [Reporting issues and contact information](#section7)
+
+##<a name="section1"></a>Batching
+<i>NOTE: The batching mechanism is currently only available for `OrionHDFSSink` and `OrionMySQLSink`.</i>
+
+Batching is the mechanism Cygnus implements for processing sets of events all together instead of one by one. These sets, or properly said <i>batches</i>, are built by `OrionSink`,  the base class all the sinks extend. Thus, having the batches already created in the inherited code the sinks only have to deal with the persistence of the data within them. Typically, the information within a whole batch is aggregated into a large data chunk that is stored at the same time by using a single write/insert/upsert operation. Why?
+
+What is important regarding the batch mechanism is it largely increases the performance of the sink because the number of writes is dramatically reduced. Let's see an example. Let's assume 100 notifications, no batching mechanism at all and a HDFS storage. It seems obvious 100 writes are needed, one per notification. And writing to disk is largely slow. Now let's assume a batch of size 100. In the best case, all these notifications regard to the same entity, which means all the data within them will be persisted in the same HDFS file and therefore only one write is required.
+
+Obviously, not all the events will always regard to the same unique entity, and many entities may be involved within a batch. But that's not a problem, since several sub-batches of events are created within a batch, one sub-batch per final destination HDFS file. In the worst case, the whole 100 entities will be about 100 different entities (100 different HDFS destinations), but that will not be the usual scenario. Thus, assuming a realistic number of 10-15 sub-batches per batch, we are replacing the 100 writes of the event by event approach with only 10-15 writes.
+
+Nevertheless, a risk when using batches is the last batch never gets built. I.e. in the above 100 size batch if only 99 events are notified and the 100th event never arrives, then the batch is never ready to be preocessed by the sink. Thats the reason the batch mechanism adds an accumulation timeout to prevent the sink stays in an eternal state of batch building when no new data arrives. If such a timeout is reached, then the batch is persisted as it is.
```

Comment:
There is another risk (that I think that it is even worse than the one related with persisting delays): it introduces state in Cygnus, so if Cygnus crashes or it is stoped, then all the accumulated batch is lost. I think that also should be mentioned in the documentation.